setup.py

```diff
@@ -84,7 +84,7 @@
             'invenio_theme_admin_js = invenio_theme.bundles:admin_js',
         ],
         'invenio_assets.webpack': [
-            'invenio_theme = invenio_theme.bundles:theme'
+            'invenio_theme = invenio_theme.webpack:theme'
```

Comment:
We need to add Flask-WebpackExt as a dependency. The [release failed](https://travis-ci.org/inveniosoftware/flask-webpackext/jobs/449870739), I will fix it and this should pass.

Reply:
Flask-WebpackExt is a dependency of invenio-assets. I don't think we need to add it here